### PR TITLE
support python3

### PR DESCRIPTION
**Pull request by gcd0318**
support python3

### CODE REVIEW

**Comment by gcd0318** [comment]
remove legacy for python 1.5.2 or earlier

**Comment by ahxxm** [comment]
does this work for you?

`NoteStore.py` is automatically generated by Thrift, I'm wondering if it's safe to directly modify it.

**Comment by gcd0318** [comment]
I'm not sure if it is perfectly fine. at least I'm using and it works. but you really enlighten me, and I will test through later. thanks for attention

File: lib/evernote/api/client.py [hunk 1]
```diff
@@ -3,8 +3,8 @@
 import inspect
 import re
 import oauth2 as oauth
-import urllib
-import urlparse
+import urllib.request, urllib.parse, urllib.error
+import urllib.parse
```
Comment (IamGianluca) [inline lib/evernote/api/client.py]: I think this could be removed since you're importing `urllib.parse` in the previous line.

**Comment by akhaku** [comment]
See https://github.com/evernote/evernote-sdk-python3 for a python3 version of our SDK

**Pull request closed without merging**